charts/yorkie-argocd/README.md

```diff
@@ -0,0 +1,93 @@
+# yorkie-argocd
+
+Installs the yorkie-argocd, which provides GitOps based continuous delivery for yorkie cluster.
+
+## Prerequisites
+
+- Kubernetes 1.24+
+- Helm 3+
+
+## Get Helm Repository Info
+
+```bash
+helm repo add yorkie https://yorkie-team.github.io/yorkie/helm-charts
+helm repo update
+```
+
+_See [`helm repo`](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Helm Chart
+
+```bash
+# Install yorkie argocd helm chart
+helm install [RELEASE_NAME] yorkie/yorkie-argocd -n argocd --create-namespace
```

Comment:
What value should I put in RELEASE_NAME?

Reply:
Anything you want to name it as alias of chart. (eg: yorkie-cluster-minikube, yorkie-cluster-aws)
For more information, follow: https://helm.sh/ko/docs/intro/using_helm/